subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -80,6 +84,33 @@ static int publish_get_payload(struct mqtt_client *client,
 	return 0;
 }
 
+static int construct_job_id_get_topic(const u8_t *client_id,
+				      const u8_t *job_id,
+				      const u8_t *suffix,
+				      u8_t *topic_buf)
+{
+	__ASSERT_NO_MSG(client_id != NULL);
+	__ASSERT_NO_MSG(topic_buf != NULL);
+	u32_t job_id_get_topic_len;
+	int ret = snprintf(topic_buf,
+			  JOB_ID_GET_TOPIC_MAX_LEN,
```

Comment:
Alignment seems off (at least on github).

Reply:
Jupp contracted it as per. @hakonfam comment